tools/state-viewer/src/commands.rs

```diff
@@ -207,8 +223,14 @@ pub(crate) fn apply_chunk(
         target_height,
         None,
         use_flat_storage,
+        use_trie_for_free,
     )?;
-    println!("resulting chunk extra:\n{:?}", resulting_chunk_extra(&apply_result, gas_limit));
+    // Most probably `PROTOCOL_VERSION` won't work if the target_height points to a time
+    // before congestion control has been introduced.
```

Comment:
Can you add a `TODO(congestion_control) Get the correct protocol version` here? Or just do it, I think you have all the data you need in place to get the version and pass it in.

Reply:
I'll try to get the data and fallback into a TODO if a fail hard 🙃